pkg/certificate/configuration.go

```diff
@@ -142,3 +144,34 @@ func (m *Manager) CABundle() ([]byte, error) {
 
 	return clientConfigList[0].CABundle, nil
 }
+
+// getServicesFromConfiguration it retrieves all the references services at
+// webhook configuration clientConfig and in case there is no service ref
+// it will refernece fake one with webhook name, default namespaces and
```

Comment:
Fixed a little the comment, discovered this after running tests with envtest we cannot use Services there since it's not a proper cluster to exercise webhooks we use URL field instead of ServiceRef.